manifests/profiles/default.yaml

```diff
@@ -288,12 +294,6 @@ spec:
         zvpn: {}
         env: {}
         meshExpansionPorts:
```

Comment:
Can we remove "meshExpansionPorts:" too?

Reply:
Oh yep, and this made me realize they also need to be removed from the subchart...

Reply:
Nevermind, will keep the subchart values because it's less likely to significantly break things that way. Removing this still